_scripts/instruction-widget/templates/getting-started/apache.html

```diff
@@ -72,6 +72,22 @@ <h4>Note:</h4>
     Certbot, Apache will automatically begin using any renewed certificates.
     </p>
 {{/installer_http01}}
+{{#dns_plugins}}
+<p>
+If you want to obtain a wildcard certificate using Let's Encrypt's new ACMEv2
+server, you'll also need to use one of
+<a href="/docs/using.html#dns-plugins">Certbot's DNS plugins</a>. To do this,
+make sure the plugin for your DNS provider is installed using the instructions
+above and run a command like the following:
+</p>
+<pre>$ sudo {{base_command}} -a dns-plugin -i apache -d *.example.com -d example.com --server https://acme-v02.api.letsencrypt.org/directory</pre>
```

Comment:
I would write `\*` instead of `*` even though people can't copypaste it directly.